File: 2.0/java/me/createApiCredential.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.account.model.ApiCredentials;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("account_login", "account_password");
        ApiCredentials credentials = new ApiCredentials();
        credentials.setName("api-integration");
        credentials = callfireClient.meApi().createApiCredentials(credentials);
    }
}

File: 2.0/csharp/me/createApiCredential.cs
using CallfireApiClient;
using CallfireApiClient.Api.Account.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("account_login", "account_password");
        ApiCredentials credentials = new ApiCredentials {Name = "api-integration"};
        credentials = client.MeApi.CreateApiCredentials(credentials);
    }
}

